test/converter/ImportEntities.test.js

```diff
@@ -13,12 +13,8 @@ const fixture = readFixture('element-citation.xml')
 
 
 test("Import journal citation", function(t) {
-  let dom = DefaultDOMElement.parseXML(fixture)
   let entityDb = _emptyEntityDb()
-  let api = _createAPI(entityDb)
-  let converter = new ImportEntities()
-
-  converter.import(dom, api)
+  _importEntities(entityDb)
```

Comment:
You can improve that, by returning an object and using spread expression to assign multiple args
```
let { entityDB, converter } = _setup(fixture) 
```